doc_source/AuroraMySQL.mysql80-upgrade-procedure.md

```diff
@@ -563,7 +563,41 @@ upgrading to avoid compatibility issues."
 
  The next sequence of examples demonstrates how to fix this particular issue and run the upgrade process again\. This time, the upgrade succeeds\. 
 
- First, we go back to the original cluster and create a new table with the same structure and contents as the one with faulty metadata\. In practice, you would probably rename this table back to the original table name after the upgrade\. 
+ First, we go back to the original cluster\. And run the `OPTIMIZE TABLE tbl_name [, tbl_name] ...` on the tables causing the Error: `Table tbl_name contains
+ dangling FULLTEXT index. Kindly recreate the table before upgrade.`.
+ 
+ See Mysql Documentation: [Optimizing InnoDB Full-Text Indexes](https://dev.mysql.com/doc/refman/5.6/en/fulltext-fine-tuning.html#fulltext-optimize) and [OPTIMIZE TABLE Statement](https://dev.mysql.com/doc/refman/5.6/en/optimize-table.html)
```

Comment:
AWS style is:

For more information, see <reference1> and <reference2> in the MySQL documentation.

Reply:
the rest of this looks fine- thanks!

Reply:
Thank you, I changed the sentence.